homeassistant/components/unifiprotect/data.py

```diff
@@ -59,6 +59,63 @@ def async_last_update_was_successful(hass: HomeAssistant, entry: ConfigEntry) ->
     )
 
 
+@callback
+def _async_debug_log_event(event: Event) -> None:
+    if not _LOGGER.isEnabledFor(logging.DEBUG):
+        return
```

Comment:
It would be a little bit faster to move this guard outside of the function so it never runs if debug logging is off